src/main/java/org/jvnet/hudson/update_center/Plugin.java

```diff
@@ -422,6 +422,11 @@ public JSONObject toJSON() throws IOException {
         if (hpi.getCompatibleSinceVersion() != null) {
             json.put("compatibleSinceVersion",hpi.getCompatibleSinceVersion());
         }
+
+        if (hpi.getMinimumJavaVersion() != null) {
+            json.put("requiredJava", hpi.getMinimumJavaVersion());
```

Comment:
In light of the recent issues with users running Java 9, or Java 10, I guess we should name this more explicitly: `minRequireJava` or something like this?

Reply:
+1